experimental/private-methods.md

```diff
@@ -0,0 +1,22 @@
+[Private class instance methods, getters and setters](https://github.com/tc39/proposal-private-methods)
+
+```js
+interface PrivateName <: Node {
+    type: "PrivateName";
+    name: string;
+}
+
+extend interface MethodDefinition {
+    key: Expression | PrivateName;
+}
+```
+
+Private methods cannot be static or have computed names.
+
+```js
+interface PrivateMemberExpression <: Expression, Pattern {
```

Comment:
Recently the [support](https://tc39.es/proposal-class-fields/#prod-OptionalChain) of optional chaining has been added to this proposal, and both `?. PrivateIdentifier` and `OptionalChain . PrivateIdentifier` have been accepted. If we are to be consistent with current [`ChainingExpression`](https://github.com/estree/estree/blob/a75c6933bcad86174d2c913eea6d1cf87db234f7/es2020.md#chainingexpression) approach in #204, does it mean that we should also add `PrivateMemberChain`?
```js
interface PrivateMemberChain extends Chain {
  type: "PrivateMemberChain";
  property: PrivateName
}
```

Personally I lean to merge `PrivateMemberChain` with `MemberChain`
```js
interface MemberChain extends Chain {
  type: "MemberChain"
  computed: boolean
  property: Expression
}
```
with an additional note that if property is a `PrivateName`, `computed` must be `false`.